nautobot_ssot_aristacv/utils/cloudvision.py

```diff
@@ -657,3 +658,18 @@ def get_ip_interfaces(client: CloudvisionApi, dId: str):
                     }
                 )
     return ip_intfs
+
+
+def get_cvp_version():
+    """Returns CloudVision portal version.
+
+    Returns:
+        str: CloudVision version from API or blank string if unable to find.
+    """
+    PLUGIN_SETTINGS = settings.PLUGINS_CONFIG["nautobot_ssot_aristacv"]
+    client = CvpClient()
```

Comment:
isn't any need to handle exceptions?

Reply:
I've added some handling for this so that it acts like the class initialization if an authentication failure occurs.